<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorDeviceMaintain</class>
 <widget class="QWidget" name="MonitorDeviceMaintain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="areaConWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>1061</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(200,200,200);
border-radius:0px;
background-color:rgb(236,236,236);</string>
   </property>
   <widget class="QPushButton" name="addPushButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
border:1px;
border-radius:5px;
background-color:rgb(18,132,194);</string>
    </property>
    <property name="text">
     <string>+添加设备</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceNameLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string>设备名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviceNameLineEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0px;
background-color:rgb(255,255,255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceNumberLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string>设备型号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviceNumberLlineEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0px;
background-color:rgb(255,255,255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="addrLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string>行政区划</string>
    </property>
   </widget>
   <widget class="QComboBox" name="addrComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0px;
background-color:rgb(255,255,255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string>状态</string>
    </property>
   </widget>
   <widget class="QComboBox" name="statusComboBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0px;
background-color:rgb(255,255,255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchPushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;</string>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QLabel" name="areaLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string>监控区域</string>
    </property>
   </widget>
   <widget class="QComboBox" name="areaComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0px;
background-color:rgb(255,255,255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="positionLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string>监控位置</string>
    </property>
   </widget>
   <widget class="QComboBox" name="positionComboBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0px;
background-color:rgb(255,255,255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="titleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1061</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:0px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(235, 235, 235, 255), stop:1 rgba(170, 170, 170, 255));</string>
   </property>
   <widget class="QLabel" name="titlePicLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Resouce.qrc">:/images/人脸采集设备信息维护小图标.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="titleNameLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>人脸采集设备信息维护</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>1061</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius:0px;
background-color:rgb(236,236,236);</string>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QWidget" name="areaButWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>1061</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(200,200,200);
border-bottom-left-radius:5px;
border-bottom-right-radius:5px;
background-color:rgb(236,236,236);</string>
   </property>
   <widget class="QPushButton" name="allSelPushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;</string>
    </property>
    <property name="text">
     <string>全选</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="allClePushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;</string>
    </property>
    <property name="text">
     <string>反选</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="deletePushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;</string>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="buttonWidget" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>3</y>
      <width>351</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
border-radius:0px;
background-color:rgb(236,236,236);</string>
    </property>
    <widget class="QPushButton" name="startPushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(200,200,200);
border-radius:2px;</string>
     </property>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="frontPushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(200,200,200);
border-radius:2px;</string>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="oneButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(200,200,200);
border-radius:2px;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="twoButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(200,200,200);
border-radius:2px;</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="threeButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(200,200,200);
border-radius:2px;</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QLabel" name="buttonLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
border-radius:0px;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="fiveButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(200,200,200);
border-radius:2px;</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextPushButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(200,200,200);
border-radius:2px;</string>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="endPushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>31</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(200,200,200);
border-radius:2px;</string>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Resouce.qrc"/>
 </resources>
 <connections/>
</ui>
